src/main/java/com/autotune/common/target/kubernetes/service/KubernetesServices.java

```diff
@@ -86,5 +86,9 @@ public interface KubernetesServices {
     //Watch endpoints. Used to trigger events based on some action to kubernetes resources.
     void watchEndpoints(CustomResourceDefinitionContext crd);
 
+    //Check if deployment is ready
     public boolean isDeploymentReady(String namespace, String deploymentName);
+
+    //Check if pods is ready
+    public boolean isPodsReady(String namespace, int podsReadyInMinutes);
```

Comment:
1. If we have to check for pods of a specific deployment, deploymentName should be one of the parameters as well?
2. Naming doesn't seem right. Should it be `isPodReady`?